site/content/docs/project.md

```diff
@@ -0,0 +1,54 @@
+# Deployment projects
+
+### What is a deployment project?
+
+A deployment project allows you to customize the resources you deploy to AWS. Instead of choosing one of the recipes supplied by AWS, you can generate a deployment project and modify it to fit your application needs.
+
+Each deployment project includes:
+
+* **.NET CDK project** - takes the collected settings from the user and performs the deployment using the AWS CDK.  It’s a standard C# console project that uses NuGet packages containing CDK construct types to define the AWS infrastructure.
+* **JSON metadata file** - a JSON configuration file that contains all of the settings that are needed to determine which deployment services to use and their configurations. Here is the [JSON file definition](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/RecipeDefinitions).
+
+### Generating a project
+
+You can generate deployment project into your workspace. The `--project-display-name` sets the name for the project that will be seen to you during the deployment.
+
+    dotnet aws deployment-project generate --project-display-name MyCustomCDKProject
+
+You can also generate the deployment project in the directory of your choice by specifying the `--output` parameter.
+
+    dotnet aws deployment-project generate --output ../myCustomCDKProject --project-display-name MyCustomCDKProject
+
+You can then customize the project and the deployment settings to fit your needs.
+
+The `AppStack class` is the recommended place to add new AWS resources or customize the resources created in the generated code.
```

Comment:
nit - This sentence would make more sense in the `Customizing Resources` section which would tell them where to customize their resources in the project.

Reply:
moved